networkx/algorithms/centrality/voterank_alg.py

```diff
@@ -90,3 +90,92 @@ def voterank(G, number_of_nodes=None):
             vote_rank[nbr][1] -= 1 / avgDegree
             vote_rank[nbr][1] = max(vote_rank[nbr][1], 0)
     return influential_nodes
+
+
+def vote_rank(G, number_of_nodes=None):
```

Comment:
We need a different name!  
How about `vote_rank_with_vote`?  Kinda long, but not that long.
Thoughts?